homeassistant/components/iammeter/__init__.py

```diff
@@ -1 +1,21 @@
 """Support for IamMeter Devices."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Component setup, do nothing."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up a config entry for iammeter."""
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, "sensor")
+    )
```

Comment:
```suggestion
    hass.config_entries.async_setup_platforms(entry, ["sensor"])
```

Or just use
```python3
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

And create `PLATFORMS = ["sensor"]`

Reply:
code changed.